<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChoixStr2Str</class>
 <widget class="QDialog" name="ChoixStr2Str">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="ReturnButton">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>310</y>
     <width>780</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
   </property>
   <property name="text">
    <string>RETURN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Base7Button">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>190</y>
     <width>120</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
   </property>
   <property name="text">
    <string>BASE 7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Base8Button">
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>189</y>
     <width>120</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
   </property>
   <property name="text">
    <string>BASE 8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Base2Button">
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>70</y>
     <width>120</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
   </property>
   <property name="text">
    <string>BASE  2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Base6Button">
   <property name="geometry">
    <rect>
     <x>674</x>
     <y>70</y>
     <width>120</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
   </property>
   <property name="text">
    <string>BASE 6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Base10Button">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>190</y>
     <width>120</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
   </property>
   <property name="text">
    <string>BASE 10</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Base12Button">
   <property name="geometry">
    <rect>
     <x>674</x>
     <y>190</y>
     <width>120</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
   </property>
   <property name="text">
    <string>BASE 12</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Base11Button">
   <property name="geometry">
    <rect>
     <x>542</x>
     <y>190</y>
     <width>120</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
   </property>
   <property name="text">
    <string>BASE 11</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Base9Button">
   <property name="geometry">
    <rect>
     <x>278</x>
     <y>190</y>
     <width>120</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
   </property>
   <property name="text">
    <string>BASE 9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Base1Button">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>70</y>
     <width>120</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
   </property>
   <property name="text">
    <string>BASE  1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Base3Button">
   <property name="geometry">
    <rect>
     <x>278</x>
     <y>70</y>
     <width>120</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
   </property>
   <property name="text">
    <string>BASE 3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Base4Button">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>70</y>
     <width>120</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
   </property>
   <property name="text">
    <string>BASE 4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Base5Button">
   <property name="geometry">
    <rect>
     <x>542</x>
     <y>70</y>
     <width>120</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
   </property>
   <property name="text">
    <string>BASE 5</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="WindowTitleEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>15</y>
     <width>280</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
border: 1px solid black;
</string>
   </property>
   <property name="text">
    <string>BASE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
